src/GenerateProgrammaticPipelineFromConfig/PipelineFromConfigCommand.php

```
@@ -0,0 +1,115 @@
+information to generate a programmatic middleware pipeline and routing
+table for your application:
+
+- The pipeline is written to config/pipeline.php.
+- The routing rules are written to config/routes.php.
+- A new configuration file, config/autoload/programmatic-pipeline.global.php,
+  is created with configuration to enable the programmatic pipeline and
+  new error handling mechanisms.
+- The script updates public/index.php to require the pipeline and
+  routing configuration files prior to running the application.
+EOT;
+
+    const HELP_OPT_CONFIG_FILE = <<< 'EOT'
+Specify a path to the configuration file; defaults to 'config/config.php'.
+The file is expected to return a PHP array value containing all
+configuration.
+EOT;
+
+    public $projectDir = '.';
+
+    /**
+     * Configure the console command.
+     */
+    protected function configure()
+    {
+        $this->setDescription('Generate a programmatic pipeline and routes from configuration.');
+        $this->setHelp(self::HELP);
+        $this->addOption('config-file', 'c', InputOption::VALUE_REQUIRED, self::HELP_OPT_CONFIG_FILE);
+    }
+
+    /**
+     * Execute console command.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int Exit status
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln(
+            '<info>Generating programmatic pipeline for an existing Expressive application...</info>'
+        );
+
+        try {
+            $generator = new Generator($output);
+            $generator->projectDir = $this->projectDir;
+            $generator->process($this->locateConfigFile($input));
+        } catch (GeneratorException $e) {
+            $console = $this->getErrorConsole($output);
+            $console->writeln('<error>Error during generation:</error>');
+            $console->writeln(sprintf('  <error>%s</error>', $e->getMessage()));
+            return 1;
```

Same: let the exception bubble up instead